server_examples/fastapi/client/http.py

```diff
@@ -111,8 +115,22 @@ def handle_response(func, response):
     "get_user": {"200": "UserResponse", "422": "HTTPValidationError"},
 }
 
-client_headers = c.additional_headers()
-client = httpx.Client(headers=client_headers)
+client_headers = c.config.additional_headers.copy()
+_limits = c.config.limits
+_transport = c.config.transport
+_client_kwargs = dict(
+    headers=client_headers,
+    timeout=c.config.timeout,
+    follow_redirects=c.config.follow_redirects,
+    verify=c.config.verify_ssl,
+    http2=c.config.http2,
+    max_redirects=c.config.max_redirects,
+)
+if _limits is not None:
```

Comment:
For all instance of this let's use the walrus operator here like this:

```python
if _limits := c.config.limits:
```

Then we can eliminate line 119 in this file.

You need to make this change in the template remember.